### DISCUSSION TITLE
Is Airflow 1.10.15 compatible with python 3.8 ?

### DISCUSSION THREAD

**Discussion by sinjup** [apache/airflow#25457] [Q&A] [answered]
Currently, my airflow 1.10.15 is working with python 3.6, I am planning to upgrade the python version to 3.8. Is Airflow 1.10.15 compatible with python 3.8 ?

**Comment by Swalloow** [comment, marked as answer]
Sure, You can use the python 3.8 constraints below.
https://github.com/apache/airflow/blob/constraints-1-10/constraints-3.8.txt
